lib/zstd.h

```diff
@@ -1077,6 +1077,14 @@ ZSTDLIB_API size_t ZSTD_sizeof_DDict(const ZSTD_DDict* ddict);
 
 typedef struct ZSTD_CCtx_params_s ZSTD_CCtx_params;
 
+typedef struct {
+    unsigned int matchPos; /* match pos in dst */
+    unsigned int offset; /* offset taking into account rep (different from seqdef) */
+    unsigned int litLength; /* literal length */
+    unsigned int matchLength; /* match length */
+    unsigned int rep; /* 0 when seq not rep and seqDef.offset otherwise */
```

Comment:
The docs aren't quite accurate in the `litLength == 0` case. Don't be afraid to add a multiline comment here.